### ISSUE TITLE
[FEATURE]: UI in KCM to configure, or at least show, which layouts are active where

### ISSUE THREAD

**Issue by phrxmd** [Bismuth-Forge/bismuth#171]
### Summary

Currently there is no single place, other than the config file, where you can see which layout is active where.

There should be an UI in the KCM to show which layout is configured in which Activity or which Virtual Desktop.

### A clear and concise description of what you want to happen

Currently when you have multiple Activities and Virtual Desktops, it's a bit of a blind flight trying to get the layouts working right everywhere - you switch to an Activity or a Virtual Desktop, you use the keyboard to select a layout that you like, but unless you memorize all of that, you never know what is configured where.

It would be nice to have an UI in the KCM that shows which layouts are active where.

This would be especially useful in conjunction with FR #170 (separate layouts per screen) in order to avoid things getting confusing.

**Comment by gikari** [comment]
> Currently there is no single place, other than the config file, where you can see which layout is active where.

I am not sure that you can see the current state of layouts' assignation in the config file.

A KCM is a Configuration module, so it can only show what's configured to be by default. The current state of the tiling is not handled by KCM in any way, and it shouldn't. Instead, it could be shown in another UI component - applet (which is not ready yet), or overlay (which is present, but it cannot be triggered just to show the info).

I think a good idea is to make a Plasma applet/widget, that shows what layout is currently active on the current surface (i.e. screen space), see Activity Widget for the example. Unfortunately the applet development is blocked by upstream issue with KWin Scripting API (See #23).

**Comment by gikari** [comment]
I reworded your request (feel free to correct). Not sure if it's possible to combine all the functionality in the tray applet, because  the applet should be big enough to show the layout name text and not be  the single icon in the menu (although it could use layout icons recently introduced).

In the worst case, two separate applets would be created, where one will be in the tray and the other would be addable by users.
